<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskNotifier</class>
 <widget class="QWidget" name="TaskNotifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Notification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblNotify">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hey!!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtTask"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnOk">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0.882157, y1:0.142, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTskDel">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0.882157, y1:0.142, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
